tests/testthat/teardown_setTestEnv.R

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
are `setup_` and `teardown_` special terms recognized by `testthat`? Can you explain how this is working to me?

Reply:
oh I see now (actually clicked the link from the PR description). Cool, cool